src/vault.py

```diff
@@ -104,6 +127,22 @@ def configure_kv_policy(self, policy: str, mount: str):
             mount_policy = fd.read()
         self._client.sys.create_or_update_policy(policy, mount_policy.format(mount=mount))
 
+    def enable_audit_device(self, device_type: str, path: str) -> None:
+        """Enable a new audit device at the supplied path."""
```

Comment:
We may add Args to docstring

Reply:
I followed the style of the file.
We could do some refactoring in a different pr.